<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>353</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Gotham Pro</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_100">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>415</width>
     <height>355</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Прямоугольник</string>
    </attribute>
    <widget class="QLabel" name="label_122">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gotham Pro</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ширина:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_102">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_121">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gotham Pro</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Длина:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1123">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gotham Pro</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ответ:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_103">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_100">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_110">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gotham Pro</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{

background:white;
border-radius:20px;
border:2px solid black;
}
</string>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Круг</string>
    </attribute>
    <widget class="QLabel" name="label_124">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>211</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gotham Pro</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Радиус круга (r):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_104">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_111">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gotham Pro</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{

background:white;
border-radius:20px;
border:2px solid black;
}
</string>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_125">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gotham Pro</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ответ:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_105">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Треугольник</string>
    </attribute>
    <widget class="QLabel" name="label_126">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gotham Pro</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Высота:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_106">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_127">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>145</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gotham Pro</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Основание:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_107">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_112">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gotham Pro</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{

background:white;
border-radius:20px;
border:2px solid black;
}
</string>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_128">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gotham Pro</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ответ:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_108">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
